crates/ironrdp-server/src/display.rs

```diff
@@ -67,10 +68,35 @@ pub struct BitmapUpdate {
     pub width: NonZeroU16,
     pub height: NonZeroU16,
     pub format: PixelFormat,
-    pub data: Vec<u8>,
+    pub data: Bytes,
     pub stride: usize,
 }
 
+impl BitmapUpdate {
+    pub fn new<D>(
+        top: u16,
+        left: u16,
+        width: NonZeroU16,
+        height: NonZeroU16,
+        format: PixelFormat,
+        data: D,
+        stride: usize,
+    ) -> Self
+    where
+        D: Into<Bytes>,
+    {
+        Self {
+            top,
+            left,
+            width,
+            height,
+            format,
+            data: data.into(),
+            stride,
+        }
+    }
+}
```

Comment:
suggestion: I would suggest we don’t add this function. I think it’s much clearer to use the struct initialization syntax, so we need to name the fields. With a function, it’s easier to confuse the parameter orders, especially since the types are u16 or NonZeroU16.

Reply:
ok